### PR TITLE
Create 0.6 user guide

### PR DESCRIPTION
**Pull request by john-light**

### CODE REVIEW

**Comment by john-light** [comment]
@Smokyish this PR is ready for review, one change I would ask for you to add is to delete all images in the docs/tutorials/images/user_guide/ folder, but leave the 06 folder untouched. Those images are for the older user guide and have all been replaced in the new guide, and I will be using this new folder structure to keep things organized going forward. (I would do this myself but the GitHub Web UI does not have a good system for deleting images in bulk and I don't want to clutter the PR with 50 commits of just deleting images one at a time.)

File: docs/videos/presentations.md [hunk 1]
```diff
@@ -1,9 +1,12 @@
 # Aragon Presentation Videos
 
+<<<<<<< HEAD
```
Comment (luisivan) [inline docs/videos/presentations.md]: Careful with this, shouldn't be here

Comment (Smokyish) [reply docs/videos/presentations.md]: thanks for the note, still working on this.

**Comment by Smokyish** [comment]
@john-light That was a question I had for you, thanks, reverting them.

**Pull request merged**